pkg/logs/client/destination.go

```diff
@@ -74,3 +88,38 @@ func (d *Destination) Send(payload []byte) error {
 
 	return nil
 }
+
+// SendAsync sends a message to the destination without blocking. If the channel is full, the incoming messages will be
+// dropped
+func (d *Destination) SendAsync(payload []byte) {
+	host := d.connManager.endpoint.Host
+	d.once.Do(func() {
+		inputChan := make(chan []byte, chanSize)
+		d.inputChan = inputChan
+		metrics.DestinationLogsDropped.Set(host, &expvar.Int{})
```

Comment:
as we have multiple senders, I am afraid that you're overriding the value multiple times.

Reply:
No because `DestinationLogsDropped` is a map and the key is the host 
(unless you put the same host several time in additional destinations)